frontend/src/index.jsx

```diff
@@ -12,16 +12,16 @@ import theme from './theme'
 
 const root = ReactDOM.createRoot(document.getElementById('root'))
 root.render(
-  <React.StrictMode>
-    <ChakraProvider theme={theme}>
-      <ColorModeScript initialColorMode={theme.config.initialColorMode} />
-      <HelmetProvider>
-        <AuthProvider>
-          <App />
-        </AuthProvider>
-      </HelmetProvider>
-    </ChakraProvider>
-  </React.StrictMode>
+  // <React.StrictMode>
```

Comment:
Sorry this is commented out on purpose or by mistake? Would appreciate explanations for such decisions (comments or PR descriptions)

Reply:
Not meant to be commented out, keep accidentally committing this, sorry